pages/docs/reference/functions.md

```diff
@@ -43,13 +43,17 @@ Stream().read() // create instance of class Stream and call read()
 
 ### Parameters
 
-Function parameters are defined using Pascal notation, i.e. *name*: *type*. Parameters are separated using commas. 
+Function parameters are defined using Pascal notation, i.e. *name*: *type*. Parameters are separated using commas.
+The Kotlin code style recommends using a [trailing comma](coding-conventions.html#trailing-commas) when you declare function parameters.
```

Comment:
This is the most questionable thing as for me, so I'll ask again: can we really recommend trailing comma in function (and class) declarations **unconditionally**?
I mean, everybody declares small functions in a way like `fun sum(a: Int, b:Int): Int` - this sentence states that it's wrong now. Why? How does trailing comma make it better?

Reply:
I don't see a contradiction here: 
Kotlin style **recommends**: recommendation != obligation

And this code line `fun sum(a: Int, b:Int): Int` is not wrong.

Reply:
I sure agree that it's not an obligation. Under "wrong" I mean "not recommended"/"contradicts with recommendations". 
The sentence says "recommend trailing comma in (any!) function parameters". So we directly say to the reader that the recommended way is different from this (my example). But this declaration is OK, isn't it?
I would expect more tailored recommendations - when is this comma useful? I really don't see a point in trailing comma in many cases (one-line declarations, for example).

Reply:
I understand the concern; thus I rewrote this part.